### ISSUE TITLE
Apple Icon

### ISSUE THREAD

**Issue by couchpilotnye** [marcaaron/ww-calc-2.0#47]
There's an Apple icon on the browser tabs. We're not using the apple image any longer. Not sure if there's something obvious to replace it with. 
![image](https://user-images.githubusercontent.com/37560660/38450603-c4f355c6-39d5-11e8-8f91-b15791408545.png)
Any suggestions? Should I ask our graphic artist?

**Comment by dentemple** [comment]
I replaced the apple logo with a Create Commons image of the state of Washington, but it's a little too pale to be permanent.

We would need either a darker image that's in the Creative Commons (which I haven't found yet), or someone would need to crop & resize the image provided in Issue #17.  (As it is right now, the width size for the map image is too long to scale down cleanly as a favicon.)

I can do the crop & resize myself if no one else wants to take it, but I only have access to Photoshop on my work comp so it'll be a few days.

**Comment by marcaaron** [comment]
Yes! Let's get the contact info for the graphic designer? I've got Photoshop as well. Maybe we can get some SVG versions of these designs and adapt them for our use to build out a version of the logo / favicon that makes sense for our purpose.

**Comment by marcaaron** [comment]
How about something like this? 

![favicon](https://user-images.githubusercontent.com/32969087/38505407-f4672b78-3bcb-11e8-878d-ba0673c651ce.png)

**Comment by marcaaron** [comment]
Would still need to create a higher res version for mobile devices.

**Comment by marcaaron** [comment]
Or this... 
![favicon2](https://user-images.githubusercontent.com/32969087/38509985-6919d216-3bd8-11e8-80bc-9b061432f047.png)

**Comment by marcaaron** [comment]
Or this... 
![fav3](https://user-images.githubusercontent.com/32969087/38510034-90420110-3bd8-11e8-8c2e-99acd571a10b.png)

**Comment by couchpilotnye** [comment]
Hi Marc, Can you invite Scott to the Slack channel? I copied him here.

**Comment by couchpilotnye** [comment]
I love all three options proposed. The size is small in the tab though, and I'd like to get Scott's input. 
![image](https://user-images.githubusercontent.com/37560660/38594332-9e3c0554-3cfa-11e8-966d-cd488e245068.png)

**Comment by marcaaron** [comment]
He's welcome to leave comments here if that's an option. But the Prog Code moderators over on Slack will have to help out with that.
